web/src/helpers/geetest.js

```diff
@@ -0,0 +1,281 @@
+/* eslint-disable */
+var document = window.document;
+var Math = window.Math;
+var head = document.getElementsByTagName("head")[0];
+
+function _Object(obj) {
+    this._obj = obj;
+}
+
+_Object.prototype = {
```

Comment:
Can you please attach a link where did you get this code?

Reply:
@oyershov https://github.com/openware/tower/blob/master/src/helpers/geetest.js